### DISCUSSION TITLE
Can we watch files in node_modules?

### DISCUSSION THREAD

**Discussion by trusktr** [yandeu/five-server#114] [Help / Questions] [answered]
I'm trying something like this:

```
five-server . --watch='dist,**/dist/,examples' --open=./examples
```

hoping that `**/dist/` will cover any dist/ folders in node_modules (I have projects `npm link`ed into node_modules). It doesn't work.

Is there a way?

**Comment by yandeu** [comment]
Unfortunately node_modules is hard coded to be ignored.
https://github.com/yandeu/five-server/blob/a7093d785366b55075248ece3a595b9a10ff55db/src/index.ts#L568

**Reply by yandeu** [reply, marked as answer]
But, you can do it programmatically:

```js
import chokidar from 'chokidar'
import pkg from 'five-server'

const { default: FiveServer } = pkg
const fiveServer = new FiveServer()

fiveServer.start({ open: './examples', watch: false })

const watcher = chokidar.watch(['**/dist/**', 'examples'])

watcher.on('all', (event, path) => {
  fiveServer.reloadBrowserWindow()
})
```

**Reply by trusktr** [reply]
Would be nice to pass something simple to CLI. That's an easy enough workaround.  Thanks!
